### ISSUE TITLE
Add deployment details

### ISSUE THREAD

**Issue by sohkai** [aragon/apm#7]
Once we've standardized the format, we should include the deployment instances of both base contracts and live instances for contracts in this repo.

**Comment by JovianBrowne** [comment]
Yes, we can add documentation for the live instances that have been deployed.

@pythonpete32 or @nivida more context is needed for this issue. 

- Has the format been standardized yet? 

- What are the deployment instances (addresses where the contracts are deployed)?

**Comment by nivida** [comment]
Moved to the backlog in Linear to answer those questions later.
